index.js

```diff
@@ -55,12 +75,14 @@ module.exports = postcss.plugin('postcss-increase-specificity', function(options
 	var opts = objectAssign({}, defaults, options);
 
 	return function(css) {
+		var status = { disabled: false };
```

Comment:
I think I would prefer this sort of setup instead of passing around `status` by reference

```js
var isPluginDisabled = false;

css.walkRules(function(rule) {
	var enableDisableResult = checkForEnableDisableComments(fromNode, toNode);
	isPluginDisabled = (enableDisableResult !== null && enableDisableResult !== undefined) ? enableDisableResult : isPluginDisabled;
        ...
});
```